docs/products/flink/concepts/important_concepts.rst

```diff
@@ -27,7 +27,17 @@ Events may still arrive after the allowed lateness for the window, so your appli
 
 For more information, see the `Apache Flink documentation <https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/dev/datastream/operators/windows/>`_.
 
-Another concept closely related to windows in Apache Flink is *watermarks*. For more information on watermarks, see the `Apache Flink documentation <https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/dev/datastream/event-time/generating_watermarks/>`_.
+
+Watermarks
+----------
+
+Another concept closely related to windows and event time in Apache Flink is *watermarks*. Flink uses watermarks as a mechanism to measure progress in event time; they flow as part of the data stream, carrying a timestamp that declares the event time reached in the data stream.
```

Comment:
I'm struggling with the following sentence, I believe that there might be a piece missing:

> they flow as part of the data stream, carrying a timestamp that declares the event time reached in the data stream

Maybe

> they flow as part of the data stream, carrying a timestamp that declares the **MINIMUM** event time reached in the data stream."

Reply:
Good point, that's now added - I also added a basic diagram (pretty much copied from the Flink documentation), which might help a bit ;)